.gitignore

```diff
@@ -0,0 +1,6 @@
+.idea/
+asset/node_modules/
+theme/css/
+theme/img/
+theme/js/
```

Comment:
This implies that the contents of the `theme/` subdirectories are not checked into the repo any longer... which raises a HUGE question for me: does that mean that all repos that deploy documentation will now need to build assets as part of the deployment?

I ask, because that may change potentially every component's travis integration...